cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -138,3 +176,61 @@ where
 		Ok(pallet_xcm::Origin::Xcm(latest_location).into())
 	}
 }
+
+/// `EnsureOriginWithArg` impl for `LocalAssetCreatorAsOwner` that
+/// a. allows signed origins
+/// b. check the asset already exists
+/// c. only the owner of the asset can create
+pub struct LocalAssetCreatorAsOwner<
```

Comment:
Since we're going to duplicate this into many AH runtimes, I'd suggest it moving it to our runtime-common crate at `bridges/snowbridge/runtime/runtime-common/Cargo.toml`.